metatron-app/metatron-discovery
# #3644 Support the aggregation type of thetaSketch in data ingestion

### Description

Support the aggregation type of thetaSketch in data ingestion

**Related Issue** : #3644 



### How Has This Been Tested?

1. Create a datasource
2. In the schema setting step, select the column with string value as measure, and select the type as Hashed Map.
3. Check the debug log to see if the measure column is changed to thetasketch type.
```
{
  "type": "thetaSketch",
  "name": "Round",
  "fieldName": "Round",
  "size": 65536,
  "shouldFinalize": true,
  "isInputThetaSketch": false,
  "errorBoundsStdDev": null
}
```

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
